nix/nixos/cardano-node-service.nix

```diff
@@ -56,12 +56,13 @@ let
           "--config ${realNodeConfigFile}"
           "--database-path ${instanceDbPath}"
           "--topology ${topology}"
-        ] ++ (lib.optionals (!cfg.systemdSocketActivation) [
+        ] ++ (lib.optionals (!cfg.systemdSocketActivation) ([
           "--host-addr ${cfg.hostAddr}"
-          "--port ${toString cfg.port}"
+          "--port ${toString (cfg.port + i)}"
```

Comment:
Shouldn't this also check `shareIpv(4|6)port`?

Reply:
no because port sharing us only supported via systemd sockets. On the command line, ports must be different owtherwise it will fail to bind.